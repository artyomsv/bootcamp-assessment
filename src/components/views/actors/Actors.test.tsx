import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import ActorsView from './index';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const result = shallow(<ActorsView actors={[]}/>);
    expect(result).toMatchSnapshot();
  });
});