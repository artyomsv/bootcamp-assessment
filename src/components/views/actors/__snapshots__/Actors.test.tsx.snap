// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<ActorsView
  actors={Array []}
  classes={
    Object {
      "content": "ActorsView-content-1",
      "toolbar": "ActorsView-toolbar-2",
    }
  }
/>
`;
